testing-internal/src/main/resources/simplelogger.properties

```diff
@@ -0,0 +1,4 @@
+org.slf4j.simpleLogger.defaultLogLevel = warn
```

Comment:
Did you intend on switching the default log level from `info` => `warn`?

Reply:
Yeah - the type of spam or resource issues we've seen would always log at warn, keeping the default at warn instead of info seems like a reasonable comproise vs having to add more `SuppressLogger`